lib/nerves_hub_web/controllers/account_controller.ex

```diff
@@ -3,9 +3,31 @@ defmodule NervesHubWeb.AccountController do
 
   alias Ecto.Changeset
   alias NervesHub.Accounts
-  alias NervesHub.Accounts.SwooshEmail
+  alias NervesHub.Accounts.{User, SwooshEmail}
```

Comment:
```suggestion
  alias NervesHub.Accounts.SwooshEmail
  alias NervesHub.Accounts.User
```
One thing we've been doing recently is actually removing alias expansions when we run into them. Its not a huge deal, but makes grepping/finding a little bit easier when future us has to poke around and change, so we opt for a few more lines as the tradeoff.

Not a huge deal, but food-for-thought

Reply:
cool, I'll keep this in mind and switch them over as I come across them